requirements.txt

```diff
@@ -13,6 +13,8 @@ django-sslify
 django-storages
 Django==1.7.9
 django_coverage
+django-shortuuidfield
+dj-database-url
```

Comment:
@danaspiegel I don't see those added in https://github.com/ORGAN-IZE/register/pull/5